device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/twin/Twin.java

```diff
@@ -276,7 +288,12 @@ public static Twin createFromDesiredPropertyJson(String json)
 
         /* SRS_TWIN_STATE_21_015: [The factory shall throw JsonSyntaxException if the JSON is invalid.] */
         /* SRS_TWIN_STATE_21_016: [The factory shall deserialize the provided JSON for the Twin class and subclasses.] */
-        Gson gson = new GsonBuilder().excludeFieldsWithoutExposeAnnotation().disableHtmlEscaping().create();
+        Gson gson = new GsonBuilder()
+                .excludeFieldsWithoutExposeAnnotation()
```

Comment:
Is there a way to centralize the common ones so you don't have so many chances for being different where not intended?

Reply:
Yeah, I can try making this a singleton instead